backend/models/user.js

```diff
@@ -915,8 +917,7 @@ function _createAccounts(roles, userName) {
 				let account = null;
 				const query = { "permissions": { "$elemMatch": { user: userName } } };
 				const projection = { "permissions": { "$elemMatch": { user: userName } }, "models": 1, "name": 1 };
-				return Project.find({ account: user.user }, query, projection).then(projects => {
-
+				return listProjects(user.user, query, projection).then(projects => {
```

Comment:
Move this logic into project. There should be a function in project to get all projects the user has access to. this is very high coupling 😆